packages/desktop-client/src/components/mobile/budget/BudgetTable.jsx

```diff
@@ -1968,12 +1986,15 @@ function MonthSelector({
         onPress={() => {
           onOpenMonthMenu?.(month);
         }}
+        aria-label="Selected budget month"
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏this one is problematic too.